### ISSUE TITLE
Network access in Natron

### ISSUE THREAD

**Issue by kirang4e** [MrKepzie/Natron#381]
Hello,
I'm verymuch new to Natron.I just downloaded and testing it on my windows 7 machine.I would say Its awesome. I can work on it directly without watching any tutorial or something.But Its getting crashed frequently when I use OFX plugins..its crashing while rendering.
I do not know ,I'm doing it right.. I can't access my data which is in another server. I tried all the ways I use in nuke.Does Natron has Network support or not? Sorry If it's dumb question.I could'nt find it in release notes. 
1 Request from my side,Please add "invert" option for masks..
Thankyou..

**Comment by MrKepzie** [comment]
> On 5 Jan 2015, at 08:28, kirang4e <notifications@github.com> wrote:
> 
> Hello,
> I'm verymuch new to Natron.I just downloaded and testing it on my windows 7 machine.I would say Its awesome. I can work on it directly without watching any tutorial or something.But Its getting crashed frequently when I use OFX plugins..its crashing while rendering.
> 

If you can describe a little bit more what is happening, maybe post a video showcasing the bug that would help.

> I do not know ,I'm doing it right.. I can't access my data which is in another server. I tried all the ways I use in nuke.Does Natron has Network support or not? Sorry If it's dumb question.I could'nt find it in release notes. 
> 

Again, where is it failing ? The file dialog ? the image reading ? A video would help 

> 1 Request from my side,Please add "invert" option for masks..
> 

We cannot add it inside the Roto node itself because of technical limitations in the OpenFX standard. Instead we had to had the “Invert mask” option for all filters.

**Comment by kirang4e** [comment]
I'm really sorry for the late reply.I was out of station,so could not reply back.Pardone me. 
May I reopen this thread?
I will address my issues one by one.
Reformat:
I want to reformat only one element in my comp. say for example,a BG matte paint which is of size 5K.Need to reformat it to 2048x872. in nuke's reformat we can create new format.
![reformat](https://cloud.githubusercontent.com/assets/10396656/5798545/3acc1600-9fee-11e4-9faf-1e3812a78a36.png)
 but in ResizeOIIO,I can not find it.It can import .nps format. so how can I define my own resolutions and import them in ResizeOIIO??

 Network Access:
Usually I access my footage from network.. I can directly paste server path in nuke's node graph.I tried same in Natron.its not working,not even in read dailogue box.. My server path is something like "//server1/proj/scene/shot1/scan/0001.exr" how to browse them?,without mapping network drives in local?.

Exporting to other platforms:
(May be this question is little weird)
Can we export roto nodes from Natron to Nuke? in future versions? 

My questions are mostly refering to nuke,I don't want to compate Natron V 1.0 to Nuke.It is still in develoment stage.but I'm just little curious :D 
I have posted these queries in facebook group and following.
